src/components/Map/Map/Map.tsx

```diff
@@ -77,64 +85,121 @@ const Map = React.forwardRef<GoogleMap, Props>((props, ref) => {
                 } else {
                     bounds.extend(firstMarker.center);
                     map.fitBounds(bounds);
                 }
-            } else if (markers.length) {
-                markers.forEach(({ center, radius }) => {
+            } else if (circularMarkers.length) {
+                circularMarkers.forEach(({ center, radius }) => {
                     if (radius) {
                         const circle = new CircleClass({ center, radius });
                         bounds.union(circle.getBounds());
                     } else {
                         bounds.extend(center);
                     }
-                    map.fitBounds(bounds);
                 });
-            } else if ('address' in defaultArea) {
-                fitBoundsByAddress(defaultArea.address);
-            } else {
-                if ('lng' in defaultArea.center) {
-                    map.setCenter(defaultArea.center);
-                } else {
-                    map.setCenter(new google.maps.LatLng(...defaultArea.center));
-                }
 
-                map.setZoom(defaultArea.zoom);
+                map.fitBounds(bounds);
+            } else {
+                centerMapToDefaultArea();
             }
         }
-    }, [defaultArea, mapRef, markers]);
+    }, [defaultArea, mapRef, circularMarkers, regionAreas]);
 
     const manageDefaultHighlight = React.useCallback(() => {
         if (mapRef && typeof mapRef !== 'function' && mapRef.current && mapRef.current.state.map) {
             const { map } = mapRef.current.state;
-            if (defaultHighlight && !markers.length && !highlightFeatures) {
+            if (defaultHighlight && !markers.length && !defaultHighligh) {
                 const highlight = map.data.addGeoJson(defaultHighlight);
-                map.data.setStyle({
-                    fillColor: 'red',
-                    strokeColor: 'red',
-                    strokeWeight: 2,
-                });
                 setHighlightFeatures(highlight);
-            } else if (markers.length && highlightFeatures) {
-                highlightFeatures.forEach((feature) => map.data.remove(feature));
+            } else if (markers.length && defaultHighligh) {
+                defaultHighligh.forEach((feature) => map.data.remove(feature));
                 setHighlightFeatures(null);
             }
         }
-    }, [defaultHighlight, highlightFeatures, mapRef, markers]);
+    }, [defaultHighlight, defaultHighligh, mapRef, markers]);
 
-    React.useEffect(fitBounds);
+    const parseMarkers = () => {
+        const cMarkers: CircularMarker[] = [];
+        const rAreas: RegionArea[] = [];
+
+        markers.forEach((item) => {
+            if ('center' in item) cMarkers.push(item);
+            if ('type' in item && 'features' in item) rAreas.push(item);
+        });
+
+        if (JSON.stringify(circularMarkers) !== JSON.stringify(cMarkers))
+            setCircularMarkers(cMarkers);
```

Comment:
Let's be consistent with the {}. If it is not set in the linter, can you add https://eslint.org/docs/rules/curly with option 'all'?

Reply:
Added this rule. Also updated some other components.